vdirsyncer/storage/ldap.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+import logging
+import ldap3
+import vobject
+
+from .base import Item, Storage
+from .. import exceptions
+
+ldap_logger = logging.getLogger(__name__)
+
+
+class LDAPStorage(Storage):
+    '''
+    :param uri: LDAP URI
+    :param search_base: search base
+    :param bind: bind dn
+    :param password: bind password
+    :param filter: filter
+    '''
+    storage_name = 'ldap'
+    read_only = True
+    fileext = '.vcf'
+    item_mimetype = 'text/vcard'
+
+    def __init__(self, uri=None, search_base=None, bind=None, password=None,
```

Comment:
I added a default value for uri to 'ldap://localhost'.
And try to get default value for base dn from server info.

-- 
Paul Fariello

PGP: 0x672CDD2031AAF49B